docs/source/user_guide/pipelines.md

```diff
@@ -107,6 +107,28 @@ Each pipeline node is configured using properties. Default node properties are a
 
  - [Disable node caching](#disable-node-caching)
 
+#### Defining pipeline parameters
+
+Certain runtime-specific pipelines include support for pipeline parameters, which are configurable in the "Pipeline Properties" panel.
```

Comment:
Not sure the panel has a formal name, but to avoid overloading the term "pipeline properties" (which is also a tab in this panel), we might want to refer to it as something else, e.g.

```suggestion
Certain runtime-specific pipelines include support for pipeline parameters, which are configurable in the "Pipeline Editor Properties" panel.
```